reference/volumes.html.md.erb

```diff
@@ -7,46 +7,51 @@ nav: firecracker
 
 <%= partial "/docs/partials/v2_transition_banner" %>
 
-Apps can store ephemeral data on the root file systems of their Machines, but a Machine's file system is rebuilt from scratch each time the app is deployed or the Machine is restarted. Volumes are local persistent storage for [Fly Machines](/docs/machines/). Volumes allow an app to save its state—to preserve configuration, session or user data—and then be restarted with that information in place.
+Apps can store only ephemeral data on the root file systems of their Machines, because a Machine's file system is rebuilt from scratch each time you deploy your app, or the Machine is restarted. Volumes are local persistent storage for [Fly Machines](/docs/machines/). You can use volumes to store your database files or backups, to save your app's state, configuration, and session or user data, or for any information that needs to persist after deploy or restart.
```

Comment:
Configuration data, session data and user data are the app's state in the original text. Just had to get that out of my system! 

```suggestion
Apps can store only ephemeral data on the root file systems of their Machines, because a Machine's file system is rebuilt from scratch each time you deploy your app, or the Machine is restarted. Volumes are local persistent storage for [Fly Machines](/docs/machines/). You can use volumes for database storage, for your app's configuration, session or user data, or for any other information that needs to persist after deploy or restart.
```

We don't have a great way to use volumes for backups, do we? You can't write to one without attaching it to a VM, and you can only attach it to one VM at a time.

Reply:
@catflydotio you're right about state. I guess I think of configuration as separate from state, but used together? But I can put it back to the way it was. :)  What do you suggest?

Reply:
I started bikeshedding on this, but whatever you want. 😆 Volumes are the persistent storage; put data on them that should persist